base/standard/src/main/java/org/openscience/cdk/geometry/GeometryUtil.java

```diff
@@ -254,6 +255,39 @@ public static void rotate(IAtomContainer atomCon, Point2d center, double angle)
         }
     }
 
+    /**
+     * Reflect a collection of atoms in the line formed by the two specified points (beg,end).
+     *
+     * @param atoms the atoms
+     * @param beg the begin point of a line
+     * @param end the end point of a line
+     */
+    public static void reflect(Collection<IAtom> atoms, Point2d beg, Point2d end) {
```

Comment:
My first thought was,  "but that changes the stereochemistry..." But then I realized that we are no longer using concepts like "wedge bond up" as model of stereochemistry :)

Reply:
Can you please add a unit test for it?

Reply:
You are right, it should flip bonds as meant to be a generic method